src/main/resources/application.yml

```diff
@@ -12,6 +12,9 @@ application:
     cognito:
       user-pool-id: ${COGNITO_USER_POOL_ID}
       dsp-consultant-group: ${COGNITO_DSP_CONSULTANT_GROUP}
+    sns:
+      user-account:
+        update: ${USER_ACCOUNT_UPDATE_EVENT_TOPIC:}
```

Comment:
I see that permissions to push to this SNS have already been added ✔️

Reply:
I remembered this time, there's a first time for everything 😆